keps/262-ConfigurableFailurePolicy/README.md

```diff
@@ -411,24 +273,12 @@ the behavior defined for each FailurePolicyAction type:
 
 1) `FailJobSet`: To fail the JobSet immediately without restarting, the controller updates the JobSet status to failed.
 
-2) `Ignore`: To "ignore" the failure (i.e., restart the JobSet without counting it against `MaxRestarts`), the controller
+2) `RestartJobSetAndIgnore`: To "ignore" the failure (i.e., restart the JobSet without counting it against `MaxRestarts`), the controller
```

Comment:
`Ignore` is fine, I don't think we should change this one. @alculquicondor did you make this suggestion?

Reply:
I did, actually. With just `Ignore`, it's not immediately obvious that one Job failure causes the restart of all the others.

`Ignore` in Job just means restart that pod.

Reply:
Yeah, I tried to come up with a more explicit enum here because I agreed with Aldo that it was a bit vague. In fact, to me `Ignore` is a bit counter-intuitive, since I think "ignore" implies "do nothing," when it in fact means "restart the entire JobSet without incrementing restart attempts."

Reply:
ok, I was assuming that we would continue to match rules down the order and only if we go through all the list without matching any, we restart jobset because it is the default behavior. 

But if the semantics is that we don't continue down the rules list, then we I agree with the name change, but perhaps we should say `RestartJobSetAndIgnoreMaxRestarts` :)

Reply:
Updated to `RestartJobSetAndIgnoreMaxRestarts`